tests/ui_tests/app/page/common/base_page.py

```diff
@@ -299,21 +276,46 @@ def assert_page_is_opened():
         with allure.step(f'Wait page {page_name} is opened'):
             wait_until_step_succeeds(assert_page_is_opened, period=0.5, timeout=timeout)
 
-    def set_locator_value(self, locator: Locator, value: str) -> None:
-        """Fill locator with value."""
+    def send_text_to_element(
+        self, locator: Locator, text: str, is_with_clear: bool = True, timeout: Optional[int] = None
```

Comment:
`is_with_clear` looks heavy, maybe just `clean_input`?
and what's the motivation to make it a flag and not decide on a test level either we want to clean input or not?

Reply:
changed